projects/angular/src/forms/styles/_textarea.clarity.scss

```
@@ -47,8 +45,19 @@
       forms-variables.$clr-forms-text-color,
       variables.$clr-use-custom-properties
     );
-    border-radius: forms-variables.$clr-forms-baseline * 0.5;
-    padding: $textarea-vertical-padding $textarea-horizontal-padding;
+    @include mixins.css-var(
+      border-radius,
+      clr-forms-textarea-border-radius,
+      forms-variables.$clr-forms-baseline * 0.5,
+      variables.$clr-use-custom-properties
+    );
+    @include mixins.css-var(
+      padding,
+      clr-forms-textarea-padding,
+      $textarea-vertical-padding $textarea-horizontal-padding,
+      variables.$clr-use-custom-properties
```

The variables are not needed any more use their values directly. Brackets are optional.

```suggestion
    (forms-variables.$clr-forms-baseline + variables.$clr_baselineRem_2px) (forms-variables.$clr-forms-baseline * 2),
      variables.$clr-use-custom-properties
```